tensor_genn/algorithms/relu_ann.py

```diff
@@ -246,15 +243,15 @@ def convert(self, tf_model, scaled_tf_weights=None):
 
         return self.g_model, self.neuron_pops, self.current_source
 
-    def evaluate(self, X, y=None, raster_plot=False):
+    def evaluate(self, X, y=None, save_example_spikes=[]):
         n_examples = len(X)
         X = X.reshape(n_examples,-1)
         y = y.reshape(n_examples)
         n = len(self.neuron_pops)
         
         n_correct = 0
-        spike_ids = [None]*len(self.neuron_pops)
-        spike_times = [None]*len(self.neuron_pops)
+        spike_ids = [[None]*len(self.neuron_pops)]*len(save_example_spikes)
+        spike_times = [[None]*len(self.neuron_pops)]*len(save_example_spikes)
```

Comment:
sadly, this doesn't work:
```python
In [5]: x= [[None]*4]*2                                                                                                         
In [6]: x                                                                                                                       
Out[6]: [[None, None, None, None], [None, None, None, None]]
In [7]: x[0][0]=1                                                                                                               
In [8]: x                                                                                                                      
Out[8]: [[1, None, None, None], [1, None, None, None]]
```
I think the best you can do is:
```python
spike_times = [[None for _ in enumerate(self.neuron_pops))] for _ in enumerate(save_example_spikes)]     
```

Reply:
Oops. I'll change it right away